### ISSUE TITLE
Suggestions for supporting other languages?

### ISSUE THREAD

**Issue by niftylettuce** [moos/wordpos#34]
Not sure if you've had to work on this @moos but curious if you or others have figured out support for multiple locales/langs.

**Comment by moos** [comment]
<del>"[WordNet](https://wordnet.princeton.edu/)® is a large lexical database of English."  wordpos is just a Javascript front to the WordNet database.</del>

Just as I was about to hit send on above, I re-thought your question and a quick Google search later came across the [Global WordNet Association
](http://globalwordnet.org/resources/wordnets-in-the-world/) -- apparently there are "WordNet"s in other languages.

I looked at a few (German, French, Japanese) -- they seem to have the data in various XML formats -- so not a dropin for wordpos which is based on a specific (optimized) WordNet format.

Still an intriguing possibility -- based mainly on level of interest and contributions.  Do you have a specific use case in mind?

**Comment by niftylettuce** [comment]
Yes, it is towards my efforts with <https://github.com/spamscanner/spamscanner>.  I am building a filter to detect gibberish and the language of the message (and not be reliant upon Content-Language or `<meta>` or `<html lang` tags.
